src/olympia/accounts/utils.py

```diff
@@ -8,17 +8,20 @@
 from django.urls import reverse
 from django.utils.encoding import force_str
 
+import olympia.core.logger
 from olympia.amo.utils import is_safe_url, use_fake_fxa
 
 
+log = olympia.core.logger.getLogger('accounts')
+
+
 def fxa_login_url(
     config,
     state,
     next_path=None,
-    action=None,
```

Comment:
I removed `action` because we were only using `signin` and this does nothing - FxA only supports `email` or `force_auth` [according to their documentation](https://mozilla.github.io/ecosystem-platform/relying-parties/reference/query-parameters#action) (and we want neither of these).